<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowReader</class>
 <widget class="QMainWindow" name="MainWindowReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Yomichan</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/img/icon_logo_32.png</normaloff>:/img/img/icon_logo_32.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QPlainTextEdit" name="textContent">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuOpenArchive">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Open from &amp;archive</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuOpenRecent">
     <property name="title">
      <string>Open &amp;recent</string>
     </property>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuOpenArchive"/>
    <addaction name="menuOpenRecent"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFindNext"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHomepage"/>
    <addaction name="actionFeedback"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuTextSize">
     <property name="title">
      <string>&amp;Zoom</string>
     </property>
     <addaction name="actionZoomIn"/>
     <addaction name="actionZoomOut"/>
     <addaction name="separator"/>
     <addaction name="actionZoomReset"/>
    </widget>
    <addaction name="menuTextSize"/>
    <addaction name="separator"/>
    <addaction name="actionToggleAnki"/>
    <addaction name="actionToggleVocab"/>
    <addaction name="actionToggleKanji"/>
    <addaction name="separator"/>
    <addaction name="actionToggleWrap"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomReset"/>
   <addaction name="separator"/>
   <addaction name="actionFind"/>
  </widget>
  <widget class="QDockWidget" name="dockVocab">
   <property name="windowTitle">
    <string>Vocabulary</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextBrowser" name="textVocabDefs">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="openLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Expression</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textVocabSearch"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockAnki">
   <property name="windowTitle">
    <string>Anki</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListWidget" name="listDefinitions"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Active tag(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboTags">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockKanji">
   <property name="windowTitle">
    <string>Kanji</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTextBrowser" name="textKanjiDefs">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="openLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Character</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textKanjiSearch"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/icon_action_open.png</normaloff>:/img/img/icon_action_open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/icon_action_quit.png</normaloff>:/img/img/icon_action_quit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit Yomichan</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/icon_action_preferences.png</normaloff>:/img/img/icon_action_preferences.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
   <property name="toolTip">
    <string>Edit preferences</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/icon_action_about.png</normaloff>:/img/img/icon_action_about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="toolTip">
    <string>About Yomichan</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/icon_action_zoom_in.png</normaloff>:/img/img/icon_action_zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/icon_action_zoom_out.png</normaloff>:/img/img/icon_action_zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionZoomReset">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/icon_action_zoom_reset.png</normaloff>:/img/img/icon_action_zoom_reset.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset zoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/icon_action_find.png</normaloff>:/img/img/icon_action_find.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Find...</string>
   </property>
   <property name="toolTip">
    <string>Find text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFindNext">
   <property name="text">
    <string>Find &amp;next</string>
   </property>
   <property name="toolTip">
    <string>Find text again</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionToggleWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Word wrap</string>
   </property>
   <property name="toolTip">
    <string>Toggle word wrap</string>
   </property>
  </action>
  <action name="actionToggleVocab">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Vocabulary</string>
   </property>
   <property name="toolTip">
    <string>Toggle definitions</string>
   </property>
  </action>
  <action name="actionHomepage">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/icon_action_homepage.png</normaloff>:/img/img/icon_action_homepage.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Homepage...</string>
   </property>
   <property name="toolTip">
    <string>Yomichan homepage</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionToggleAnki">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Anki</string>
   </property>
  </action>
  <action name="actionFeedback">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/img/icon_action_feedback.png</normaloff>:/img/img/icon_action_feedback.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Feedback...</string>
   </property>
  </action>
  <action name="actionToggleKanji">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Kanji</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionToggleVocab</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockVocab</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowReader</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggleAnki</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockAnki</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggleKanji</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockKanji</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
